src/VirtualMonoRepo/source-mappings.json

```diff
@@ -154,9 +133,6 @@
             "name": "source-build-externals",
             "defaultRemote": "https://github.com/dotnet/source-build-externals",
             "exclude": [
-                "src/application-insights/**/*.exe",
```

Comment:
I think we should leave these exclusions.  We don't require these today.  SBE is only used in source-only builds so there is no need to have these binaries in the VMR.

Reply:
Fixed in https://github.com/dotnet/installer/pull/18563/commits/7849f84f89d000e707eb1dc2a463cd55beeb5c45